blueman/main/DBusProxies.py

```diff
@@ -8,7 +8,7 @@ class DBusProxyFailed(Exception):
 
 
 class ProxyBase(Gio.DBusProxy, metaclass=SingletonGObjectMeta):
-    def __init__(self, name, interface_name, object_path='/', systembus=False, *args, **kwargs):
+    def __init__(self, name, interface_name, object_path='/', systembus=False, flags=0, *args, **kwargs):
```

Comment:
Shouldn't the value better be `Gio.DBusProxyFlags.NONE`?

Reply:
I am saving a bit of space, ``Gio.DBusProxyFlags.NONE`` is 0.